core/config/toml/types.go

```diff
@@ -1478,16 +1480,63 @@ func (drl *DispatcherRateLimit) setFrom(f *DispatcherRateLimit) {
 	}
 }
 
+type GatewayConnector struct {
+	ChainIDForNodeKey         *string
+	NodeAddress               *string
+	DonID                     *string
+	Gateways                  []ConnectorGateway
+	WsHandshakeTimeoutMillis  *uint32
```

Comment:
I think `WS` should be upper case according to Go naming rules:
```suggestion
	WSHandshakeTimeoutMillis  *uint32
```